lib/engine/game/g_1873/step/dividend.rb

```diff
@@ -155,6 +155,10 @@ def payout_shares(entity, revenue)
             @log << "#{entity.owner.name} receives #{@game.format_currency(revenue)}"
             @game.bank.spend(revenue, entity.owner)
           end
+
+          def half_pay_withhold_amount(_entity, revenue)
+            revenue / 2
```

Comment:
does this work in ruby and javascript?

Reply:
It should (at least for 1873) since the revenue is always even.